### PR TITLE
Fix docs to allow for proper doc generation

### PR DESCRIPTION
**Pull request by bsarabeswaran**
Fixed some formatting and added toolchest.filesystem as a submodule

### CODE REVIEW

**Review by jguiditta — APPROVED** [review]
I built the docs locally with this change, and can verify they now build as expected.

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -117,6 +117,9 @@ Before you submit a pull request, check that it meets these guidelines:
    a. Make sure that there is an :code:`__init__.py`` in the same folder as the submodule.
    b. In :code:`docs/source/` add a file named :code:`toolchest.<submodule_path>.rst` and
       make it the same format as the peer files there.
+   c. You may also need to add the submodule to :code:`docs/source/toolchest.rst`or
```
Comment (jpichon) [inline CONTRIBUTING.rst:120]: (Nit) A space is missing after rst` there (it also [doesn't display correctly](https://github.com/bsarabeswaran/toolchest/blob/fix_doc_review/CONTRIBUTING.rst#pull-request-guidelines) without it)

**Review by jpichon — COMMENTED** [review]
Noted a typo inline, lgtm otherwise!

**Review by jpichon — APPROVED** [review]
Thanks for the update!

**Pull request merged**